docs/content/en/docs/resources/_index.md

```diff
@@ -37,5 +38,12 @@ description: "Additional resources to learn about Tetragon"
 
 ### Hands-on lab
 
-[Security Observability with eBPF and Tetragon](https://isovalent.com/labs/) - Natália Réka Ivánkó, Roland Wolters, Raphaël Pinson
+[Getting Started with Tetragon](https://isovalent.com/labs/tetragon-getting-started/) - Natália Réka Ivánkó, Roland Wolters, Raphaël Pinson
```

Comment:
Thanks!! as a follow up maybe we could add more of those since we are now linking to one labs (not the whole laboratory)